arrow/arrow_test.go

```diff
@@ -1,11 +1,15 @@
 package arrow_test
 
 import (
+	"strconv"
 	"testing"
 
 	"github.com/apache/arrow/go/arrow/array"
 	"github.com/apache/arrow/go/arrow/math"
 	"github.com/apache/arrow/go/arrow/memory"
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/flux/arrow"
+	mem "github.com/influxdata/flux/memory"
```

Comment:
In other files Jonathan has been adopted the convention of calling the `arrow/memory` package `arrowmemory` and not aliasing the `flux/memory` package.  It might be good to follow that here too.